tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright StackHPC, 2018
+#
+---
+- name: Install Infiniband and Gluster packages
+  package:
+    name: "{{ item }}"
+    state: present
+  become: yes
+  with_items:
+    - '@InfiniBand Support'
+    - centos-release-gluster
```

Comment:
Does this not need to be in a separate task from `glusterfs-server`? Also, looks like it requires EPEL (which would be another task).

Reply:
It looks like centos-release-gluster comes from CentOS Extras, not EPEL.

Reply:
Right you are - epel isn't required.

Reply:
I think you're right @oneswig, however EPEL was already installed in the Centos image I deployed, so I'm not sure if we've used any packages from it or not.